### DISCUSSION TITLE
How Do I Disable Raw Photos *and* Select a Resolution?

### DISCUSSION THREAD

**Discussion by Zoidmania** [gopro/labs#975] [Q&A] [answered]
# Background

I'm doing a "construction timelapse". I have a Hero 12 configured like so:

- Configure settings: `mPr5XfW0dPmPRoW0g0q0v0oV0oB1oD4oC0oS0oL0oR1*TUSB=1*DSPL=300` ![image](https://github.com/gopro/labs/assets/6587312/d83101df-4913-428c-9d5a-74fdd1e6bc5d)
- Configure intermittent photo action: `>t:W1<t:W6>07:59<18:01!60S~!08:00N!1R` ![image](https://github.com/gopro/labs/assets/6587312/320abbfe-ab87-4fe4-af93-f54e2454e682)

It's running on wall power via USB-C, with the internal battery removed (though I'd also like adivce re: whether that's sensible). In my environment, I can't use WiFi or Bluetooth, so the Hero 12 has a 1TB SD card (a SanDisk model from the recommended lists) for local storage. I'd like to leave it undisturbed for as long as possible before I have to offload the photos.

# The Problem

This configuration emits two files per photo, a JPEG and a raw. I have no need for the raw files, so I'd like to prevent their creation if possible. I only want JPEGs since local storage space is a concern.

# My Attempts to Research

On [this page](https://gopro.github.io/labs/control/custom/), with "Photo Mode" checked, the "Photo Properties" section suggests that "JPEG only" is possible, and the resulting command is `mPr0` with "JPEG only" checked. However, I have `r5X` in my settings (I'd like 5k 8:7, so this is purposeful). I don't see `r0` listed on the [settings page](https://gopro.github.io/labs/control/settings/), but I'm assuming that `r0` would contradict `r5X`. Is that correct?

I haven't found anything about this specifically on these forums (though they've been tremendously helpful so far).

**Comment by dnewman-gpsw** [comment, marked as answer]
This is a lot wrong in your first QR Code. You can't set photo resolution (yet you are trying to.)

Just set up the features you need, don't set everything.

This is all you needed in the first QR (mode Photo, Raw off.)
![image](https://github.com/gopro/labs/assets/10856084/9a89905a-4edf-4209-94a2-c4f57ede2b09)

**Reply by Zoidmania** [reply]
Thank you, I didn't realize that resolution can't be set in photo mode.
